src/Components/Constants.tsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import ms from "ms";
+var locale = require("browser-locale")();
+import humanize from "humanize-duration";
+import { Locale } from "./Types";
+import { createMuiTheme } from "@material-ui/core/styles";
+
+interface Constants {
+  classes: any;
+  commonlyUsedText: number[];
+  quickSelectIntervals: number[];
+  relativeIntervals: number[];
+  theme: any;
+  minimumYearValue: number;
+  maximumYearValue: number;
+  localeObj: Locale;
+}
+
+export const GlobalContext: React.Context<Constants> = React.createContext({
+  classes: null,
```

Comment:
hmm didnt know that was an issue

Reply:
Less an issue and more "but why?"  Doesn't that mean that the variable's only allowable value is `null`?  Did you mean `any`?

Reply:
It's not a type annotation, so I can't assign it to any... I assigned it to null because I didn't know a good init value for the makeStyles hook. Is there a better practice?

Reply:
Done